Tests/scripts/collect_tests/collect_tests.py

```diff
@@ -531,6 +532,10 @@ def _collect_pack(
             if (not allow_incompatible_marketplace or (allow_incompatible_marketplace and not is_xsoar_and_xsiam_pack)) \
                     and not collect_only_to_upload:
                 raise
+        finally:
+            # If changes are done to README files. Upload only.
+            if reason == CollectionReason.README_FILE_CHANGED:
+                collect_only_to_upload = True
```

Comment:
```suggestion

        # If changes are done to README files. Upload only.
        if reason == CollectionReason.README_FILE_CHANGED:
            collect_only_to_upload = True
```
It's not related to the try/except